internal/backend/usageupdater/config.go

```diff
@@ -0,0 +1,26 @@
+package usageupdater
+
+import (
+	"time"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/configset"
+)
+
+type Config struct {
+	Enabled  bool          `koanf:"enabled"`
+	Endpoint string        `koanf:"endpoint"`
+	Interval time.Duration `koadnf:"interval_seconds"`
+}
+
+var (
+	Configs = configset.Set[Config]{
+		Default: &Config{
+			Enabled:  true,
+			Endpoint: "http://localhost:9980",
```

Comment:
i would put in dev for now, and disable default until we have the real thing

Reply:
done